### PR TITLE
Task 6, part 1 (push notifications)

### PR DESCRIPTION
**Pull request by sonata08**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -12,7 +12,7 @@ android {
 
     defaultConfig {
         applicationId "com.example.myprofile"
-        minSdk 24
+        minSdk 26
```
Comment (1937Carrie) [inline app/build.gradle:15]: What is the reason for upgrading minSdk version?

**Review by 1937Carrie — APPROVED** [review]
Interview question:
First video is screen recording of android studio emulator. You may notice heads-up notification was appeared as expected. Second video was recorded on real device, my own phone. Why did not the heads-up notification appear as expected? Feel free to ask any additional questions.

https://github.com/user-attachments/assets/e1636c77-d611-42fa-830a-fadbe8e54981


https://github.com/user-attachments/assets/48604486-d9e4-4661-86d1-f53c5178ae39

File: app/build.gradle (hunk 1 above)
Comment (sonata08) [reply app/build.gradle:15]: changed minSdk just for simplicity for this app to reduce the amount of “if”. I understand the consequences for real projects

**Comment by sonata08** [comment]
The notification might not be showing as a heads-up due to the app settings, where showing notifications as a heads-up was blocked.

**Comment by 1937Carrie** [comment]
> The notification might not be showing as a heads-up due to the app settings, where showing notifications as a heads-up was blocked.

But I have made first launch immediately after app was installed. I didn't change manually notification visibility settings. If this is needed, I can provide video screen record another phone, where notification appears as heads-up without any manipulations in app settings.

**Pull request merged**